Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -67,6 +67,13 @@ private static InputActionAsset CreateFromJson(AssetImportContext context)
                 // Attempt to parse JSON
                 asset.LoadFromJson(content);
 
+                // If this JSON was authored before we switched to enum-by-value, migrate it now
```

Comment:
This is handled by LoadFromJson (above) now and should no longer be part of importer right?